plugin.video.themoviedb.helper/addon.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.themoviedb.helper"
+version="2.4.36"
+name="TheMovieDb Helper"
+provider-name="jurialmunkey">
+<requires>
+  <import addon="xbmc.python" version="3.0.0"/>
+  <import addon="script.module.simplecache" version="1.0.11" />
+  <import addon="script.module.requests" version="2.9.1"/>
+  <import addon="script.module.pil" version="1.1.7"/>
+</requires>
+<extension point="xbmc.python.pluginsource" library="plugin.py">
+  <provides>video</provides>
+</extension>
+<extension point="xbmc.service" library="service.py" />
+<extension point="xbmc.python.script" library="script.py" />
+<extension point="kodi.context.item">
+  <menu id="kodi.core.main">
```

Comment:
@basrieter I thought now we need to use args?

Reply:
nevermind, those are different files for each menu item

Reply:
They could now use a single library in combination with arguments. See here for an example: https://github.com/retrospect-addon/plugin.video.retrospect/blob/kodi-matrix/addon.xml. 

Especially here it would simplify the code and reduce the number of 'passthrough` python files. All the `context-*.py` could be put in a single file. I would say that would be preferred.

Reply:
I didn't even know this was possible - is this a Matrix addition? My only concern would be that I'm trying to keep things compatible across Leia/Matrix as much as possible (at least for the time being) to minimize how much work needs to be done on a separate branch. However, if it works in Leia too, then I'll definitely change to this approach because it is much better.

Reply:
Yeah Matrix only, I'd say to keep it like it is until you ditch leia :)

Reply:
After we included this in Matrix, I created at patch that I apply to the Leia version to get a Matrix version with these menu change.

Reply:
I decided to bite the bullet and make the change. I doubt I'm going to add many if any more context menu entries - and if I do, the merge between branches should be relatively clean.